lib/matplotlib/tests/test_mathtext.py

```diff
@@ -333,6 +333,9 @@ def test_operator_space(fig_test, fig_ref):
     fig_test.text(0.1, 0.6, r"$\operatorname{op}[6]$")
     fig_test.text(0.1, 0.7, r"$\cos^2$")
     fig_test.text(0.1, 0.8, r"$\log_2$")
+    # Check spacing after Sub/Super scripts
+    # Github issue 17852
+    fig_test.text(0.1, 0.7, r"$\sin^2 \cos$")
```

Comment:
Should one position the text elsewhere? Although it may not be the case here, if one continue to reuse coordinates sooner or later there will be a black area...

Reply:
Yes, of course.